README.md

```diff
@@ -195,6 +195,26 @@ Install custom reporters as a development dependency and you can use the `--repo
 nyc report --reporter=<custom-reporter-name>
 ```
 
+## Setting the project root directory
+
+nyc runs a lot of file system operations relative to the project root directory.
+During startup nyc will look for the *default* project root directory.
+The *default* project root directory is the first directory found that contains a `package.json` file when searching from the current working directory up.
+If nyc fails to find a directory containing a `package.json` file, it will use current working directory.
+You can change the project root directory with the `--cwd` option.
+
+**Note:** nyc stanza configuration is taken from the `package.json` file in the *default* project root directory.
```

Comment:
Configuration can also be taken from `.nycrc`, `.nycrc.json` or `nyc.config.js` found in the default project root directory.

Reply:
Ok, they deserve a mention here too.  I'll make the change.

Reply:
For now lets remove mention of loading the configuration from this section.  Configuration loading / precedence can be documented in a section about configuration (in a separate PR).